streaming/streaming_internal_test.go

```diff
@@ -87,8 +88,8 @@ func TestReceivedUpdate(t *testing.T) {
 			args: args{
 				u: Update{Sequence: 2,
 					TradeUpdates: []*TradeUpdate{
-						{Base: decimal.NewFromFloat64(0.01, 2),
-							Counter: decimal.NewFromFloat64(0.001, 2), OrderID: "1"},
+						{Base: decimal.NewFromFloat64(0.02, 2),
```

Comment:
Please add a test case to check that if no updates are received, LastTrade is zero.

Reply:
TestOrderBook(), makes that check if I am not mistaken?